server/src/test/java/io/crate/metadata/doc/DocTableInfoTest.java

```diff
@@ -224,9 +220,22 @@ public void test_can_retrieve_all_parents_of_nested_object_column() throws Excep
 
         TableInfo table = e.resolveTableInfo("tbl");
         Iterable<Reference> parents = table.getParents(new ColumnIdent("o1", List.of("o2", "x")));
-        assertThat(parents, contains(
+        assertThat(parents).contains(
             table.getReference(new ColumnIdent("o1", "o2")),
             table.getReference(new ColumnIdent("o1"))
-        ));
+        );
+    }
+
+    @Test
+    public void test_version_created_is_read_from_partitioned_template() throws Exception {
```

Comment:
Do we also have a test to check that `Version.CURRENT` is used without providing customSettings?